src/common/tracer.cc

```diff
@@ -55,6 +65,50 @@ bool Tracer::is_enabled() const {
   return g_ceph_context->_conf->jaeger_tracing_enable;
 }
 
+void encode(const jspan_context& span_ctx, bufferlist& bl, uint64_t f) {
```

Comment:
what about information other than the span id and trace id?
tags? baggabe?

Reply:
tags are not supposed to be propagated into child spans. baggage actually does. but I thought it isn't necessary right now.
but I think that it probably would be a problem to add later baggage encode/decode support since the baggage size is not constant. so we will need to encode the information separately.

Reply:
yes. we should probably add that up front

Reply:
as discussed. we would descope baggage de/serialization to when we actually use it